app/views/pages/help.html.slim

```diff
@@ -2,80 +2,193 @@
 
 h1 = t('titles.help')
 
-h2 = t('headings.help.creating_account.title')
+h3 = t('pages.help.what_is_login_gov.question_1', app: APP_NAME)
 
-h3 = t('headings.help.creating_account.confirmation_email')
+p = t('pages.help.what_is_login_gov.answer_1', app: APP_NAME)
 
-p = t('pages.help.creating_account.confirmation_email')
+p = t('pages.help.what_is_login_gov.answer_2_html',
+      app: APP_NAME,
+      general_services_administration_link: link_to(t('pages.help.what_is_login_gov.general_services_administration_link'), 'https://gsa.gov'))
```

Comment:
For consistency's sake should we open this and other external links in new windows?